.travis.yml

```diff
@@ -26,10 +26,10 @@ install:
   - conda install -c conda-forge aipy
   - pip install coveralls
   - pip install git+https://github.com/HERA-Team/pyuvdata.git
-  - pip install git+https://github.com/HERA-Team/hera_cal.git
+  - python setup.py install  # Needs to be installed before hera_cal
```

Comment:
Yes, I like the idea of burying the import in hera_cal until we need it. I can make that change (either before or after we finish this PR)